<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\pbazan\Documents\Visual Studio 2010\Projects\TestUnits\IntegrationTests\bin\Release\IntegrationTests.dll" total="1" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2012-02-01" time="09:24:03">
  <environment nunit-version="2.5.10.11092" clr-version="2.0.50727.4963" os-version="Microsoft Windows NT 6.1.7600.0" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.5.10\bin\net-2.0" machine-name="ABP-PC-002" user="pbazan" user-domain="ABPRO" />
  <culture-info current-culture="es-MX" current-uiculture="es-ES" />
  <test-suite type="Assembly" name="C:\Users\pbazan\Documents\Visual Studio 2010\Projects\TestUnits\IntegrationTests\bin\Release\IntegrationTests.dll" executed="True" result="Failure" success="False" time="0.066" asserts="0">
    <results>
      <test-suite type="Namespace" name="IntegrationTests" executed="True" result="Failure" success="False" time="0.054" asserts="0">
        <results>
          <test-suite type="TestFixture" name="FileHandlerTests" executed="True" result="Failure" success="False" time="0.053" asserts="0">
            <results>
              <test-case name="IntegrationTests.FileHandlerTests.CreateFileDirectoryNotFound" executed="True" result="Failure" success="False" time="0.036" asserts="0">
                <failure>
                  <message><![CDATA[System.IO.DirectoryNotFoundException was expected]]></message>
                  <stack-trace />
                </failure>
              </test-case>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>